prisma/migrations/20220214103944_add_auditing_to_users_table/migration.sql

```diff
@@ -0,0 +1,32 @@
+-- Add User_Audit table along with trigger for this table
+CREATE TABLE "User_Audit"
+(
+    operation char(20) NOT NULL,
+    timestamp timestamp DEFAULT CURRENT_TIMESTAMP,
+    id        text     NOT NULL,
+    before    JSONB    NOT NULL,
+    after     JSONB    NOT NULL
+);
+
+CREATE OR REPLACE FUNCTION User_Audit_Trigger() RETURNS TRIGGER AS
+$User_Audit$
+BEGIN
+    IF (TG_OP = 'DELETE') THEN
+        INSERT INTO "User_Audit" (operation, id, before, after)
+        VALUES ('DELETE', old.id, TO_JSONB(old), '"null"');
```

Comment:
Are `old` & `new` keywords? Wondering when we set the value for us to access the ids for the insert statements?